src/js/components/Button/Button.js

```diff
@@ -402,6 +438,42 @@ const Button = forwardRef(
       contents = <Badge content={badgeProp}>{contents}</Badge>;
     }
 
+    if (busy || success) {
+      // match what the label will use
+      let animationColor;
+      if (kind) {
+        if (!plain) {
+          animationColor =
+            (hover && getIconColor(themePaths.hover, theme)) ||
+            getIconColor(themePaths.base, theme, color, kind);
+        }
+      } else if (primary) {
+        animationColor =
+          theme.global.colors.text[isDarkBackground() ? 'dark' : 'light'];
+      }
+
+      contents = (
+        // position relative is necessary to have the animation
+        // display over the button content
+        <Box style={{ position: 'relative' }}>
```

Comment:
Maybe do `const RelativeBox = styled(Box)``position: relative``;` rather than inline style?

Reply:
Fixed in latest commit